dist/img/clip-line.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="15.23" height="13.679" viewBox="0 0 15.23 13.679"><defs><style>.a{fill:#a3a3a3;stroke:#a3a3a3;stroke-width:0.4px;}</style></defs><path class="a" d="M10.89,9.814a.37.37,0,0,1,0,.53.384.384,0,0,1-.538,0L7.291,7.354l-.013-.013A1.4,1.4,0,0,0,5.9,6.82a.98.98,0,0,0-.378.2.847.847,0,0,0-.235.341,1.28,1.28,0,0,0,.387,1.3l5.567,5.485.022.022a2.219,2.219,0,0,0,2.288.657,1.94,1.94,0,0,0,.706-.416,1.744,1.744,0,0,0,.444-.674,2.5,2.5,0,0,0-.871-2.556c-.853-.845-1.657-1.711-2.453-2.565A39.226,39.226,0,0,0,6.34,3.808a3.231,3.231,0,0,0-2.51-.578A3.308,3.308,0,0,0,2.794,3.6a2.677,2.677,0,0,0-.8.674,2.351,2.351,0,0,0-.44,1.93,5.271,5.271,0,0,0,1.884,2.92l.28.276,2.866,2.85.969.963a.374.374,0,0,1,0,.53.388.388,0,0,1-.538,0l-.969-.963L3.185,9.928,2.91,9.656a5.886,5.886,0,0,1-2.1-3.318A3.073,3.073,0,0,1,1.4,3.822a3.447,3.447,0,0,1,1.017-.867A3.947,3.947,0,0,1,3.7,2.5a3.991,3.991,0,0,1,3.11.722,39.812,39.812,0,0,1,5.132,4.9c.778.84,1.568,1.69,2.435,2.543l.009.009a3.185,3.185,0,0,1,1.04,3.322,2.5,2.5,0,0,1-.631.967,2.661,2.661,0,0,1-.991.587,2.972,2.972,0,0,1-3.075-.845s-.013-.009-.018-.018L5.14,9.2l-.013-.013A1.989,1.989,0,0,1,4.581,7.1a1.592,1.592,0,0,1,.444-.63,1.636,1.636,0,0,1,.684-.363,2.146,2.146,0,0,1,2.137.735l3.048,2.981Z" transform="translate(-0.551 -2.228)"/></svg>
```

Comment:
if you don't use this file, just remove it?

Reply:
finished